web-ui/src/main/resources/catalog/components/search/searchmanager/SearchFormDirective.js

```diff
@@ -123,9 +123,9 @@
 
       $scope.searching++;
       $scope.searchObj.params = angular.extend({},
-          $scope.searchObj.defaultParams || defaultParams,
```

Comment:
@fxprunayre I think you did the original change, but was not very clear the reason as gives priority t default search params over the actual search params, seem.

@juanluisrp has changed it to give priority to actual search params over the default ones, what seem as should be, unless we're missing anything. 

Can you review this?